src/pages/welcome/stepTwo/index.js

```diff
@@ -34,33 +37,33 @@ const StepTwo = ({ data, setData, setCanProgress, message, setMessage }) => {
   return (
     <>
       <div className="welcome-formheader">
-        <h3>Contact Info</h3>
+        <h3>{t('contactInfo')}</h3>
       </div>
       <Form className="welcome-form">
         <div className="welcome-form-inputs">
           <TextInput
             onChange={onInput}
             value={data.email}
             name="email"
-            label={"Email*"}
+            label={`${t('email')}*`}
```

Comment:
I believe the other changes in other components so far have been "email *" (with a space). Could you double check what's already been done?

Reply:
Done